README.md

```diff
@@ -365,16 +365,18 @@ productType,name.en,variantId
 
   Options:
 
-    -h, --help                 output usage information
-    -t, --template <file>      CSV file containing your header that defines what you want to export
-    -o, --out <file>           Path to the file the exporter will write the resulting CSV in
-    -j, --json                 Export in JSON format
-    -q, --queryString <query>  Query string to specify the sub-set of products to export
-    -l, --language [lang]      Language used on export for localised attributes (except lenums) and category names (default is en)
-    --queryType <type>         Whether to do a query or a search request
-    --queryEncoded             Whether the given query string is already encoded or not
-    --fillAllRows              When given product attributes like name will be added to each variant row.
-    --categoryBy               Define which identifier should be used to for the categories column - either slug or externalId. If nothing given the named path is used.
+    -h, --help                    output usage information
+    -t, --template <file>         CSV file containing your header that defines what you want to export
+    -o, --out <file>              Path to the file the exporter will write the resulting CSV in
+    -j, --json                    Export in JSON format
+    -q, --queryString <query>     Query string to specify the sub-set of products to export
+    -l, --language [lang]         Language used on export for localised attributes (except lenums) and category names (default is en)
+    --queryType <type>            Whether to do a query or a search request
+    --queryEncoded                Whether the given query string is already encoded or not
+    --fillAllRows                 When given product attributes like name will be added to each variant row.
+    --categoryBy                  Define which identifier should be used to for the categories column - either slug or externalId. If nothing given the named path is used.
+    --filterVariantsByAttributes  Query string to filter variants of products
+    --filterPrices  Query string to filter prices of variants
```

Comment:
The 2 filters are not added to the query, but are applied to the result. So yes they can be chosen independently.